src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -61,6 +66,20 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig, gdDAO: Datap
     }
   }
 
+  def processClusterRequest(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest): ClusterRequest = {
+    // create a LabelMap of default labels
+    val defaultLabels = DefaultLabels(clusterName, googleProject, clusterRequest.bucketPath, clusterRequest.serviceAccount, clusterRequest.jupyterExtensionUri)
+      .toJson.asJsObject.fields.mapValues(labelValue => labelValue.convertTo[String])
+    val defLabels2 = DefaultLabels(clusterName, googleProject, clusterRequest.bucketPath, clusterRequest.serviceAccount, clusterRequest.jupyterExtensionUri)
+      .toJson.asJsObject.fields.mapValues(labelValue => labelValue.convertTo[String])
+    // combine default and given labels
+    val allLabels = clusterRequest.labels ++ defLabels2
```

Comment:
Copy-pasting from outdated comment - 

Looking at the ticket, it specifies that user labels should not overwrite the defaults.
But I guess the next point of discussion would be whether if the user does specify a default label, should we just silently override it with our own (like it's doing in my code), or should it throw an error saying you can't even attempt to input a label with a default label key?

Reply:
I'm OK with the former but we should document the behaviour in swagger.

Reply:
And a code comment